src/server.cpp

```diff
@@ -1051,6 +1050,14 @@ PlayerSAO* Server::StageTwoClientInit(session_t peer_id)
 		return NULL;
 	}
 
+	// Player already joined: Redirect the peer ID
+	if (player->getPeerId() != peer_id) {
```

Comment:
Would it make sense to add a log message here? It could potentially help with tracking when exactly a player was ghosted, and could help detect stuff like brute-forcing.

Reply:
Added